--- quickaccess-0.7.1/quickaccess.h~	2008-08-03 12:29:26.000000000 +0200
+++ quickaccess-0.7.1/quickaccess.h	2009-01-01 13:15:10.000000000 +0100
@@ -22,7 +22,7 @@
 
 //Plasma includes
 #include <Plasma/Applet>
-#include <plasma/widgets/icon.h>
+#include <Plasma/IconWidget>
 
 //locale includes
 #include "popupdialog.h"
@@ -63,7 +63,7 @@
     
   private:
     Settings *m_settings;
-    Plasma::Icon *m_icon;
+    Plasma::IconWidget *m_icon;
     PopupDialog *m_dialog;
     QSize m_dialogSize;
     bool m_dragOver;
--- quickaccess-0.7.1/quickaccess.cpp~	2008-08-03 12:29:26.000000000 +0200
+++ quickaccess-0.7.1/quickaccess.cpp	2009-01-01 13:16:34.000000000 +0100
@@ -40,7 +40,7 @@
 QuickAccess::QuickAccess(QObject *parent, const QVariantList &args)
   :Plasma::Applet(parent, args)
   ,m_settings(new Settings(this))
-  ,m_icon(new Plasma::Icon(this))
+  ,m_icon(new Plasma::IconWidget(this))
   ,m_dialog(0)
   ,m_dialogSize(QSize())
   ,m_dragOver(false)
--- quickaccess-0.7.1/CMakeLists.txt.orig	2009-02-19 23:38:03.000000000 +0100
+++ quickaccess-0.7.1/CMakeLists.txt	2009-02-19 23:38:37.000000000 +0100
@@ -4,7 +4,6 @@
 # Find the required Libaries
 find_package(KDE4 REQUIRED)
 include(KDE4Defaults)
-find_package(Plasma REQUIRED)
  
 add_definitions (${QT_DEFINITIONS} ${KDE4_DEFINITIONS})
 include_directories(
@@ -21,10 +20,10 @@
 kde4_add_ui_files(quickaccess_SRCS quickaccessConfig.ui pluginwidgetbase.ui)
 kde4_add_plugin(plasma_applet_quickaccess ${quickaccess_SRCS})
 target_link_libraries(plasma_applet_quickaccess 
-                      ${PLASMA_LIBS} ${KDE4_KDEUI_LIBS} ${KDE4_KIO_LIBS} kfile konq)
+    ${KDE4_PLASMA_LIBS} ${KDE4_KDEUI_LIBS} ${KDE4_KIO_LIBS} kfile konq)
  
 install(TARGETS plasma_applet_quickaccess
         DESTINATION ${PLUGIN_INSTALL_DIR})
  
 install(FILES plasma-applet-quickaccess.desktop
-        DESTINATION ${SERVICES_INSTALL_DIR})
\ No newline at end of file
+        DESTINATION ${SERVICES_INSTALL_DIR})
